test/test_prototype_builtin_datasets.py

```diff
@@ -35,6 +45,15 @@ def test_coverage():
         )
 
 
+def extract_datapipes(dp):
```

Comment:
Is this the same as https://github.com/pytorch/pytorch/blob/40bb880009e86831d5186f619b33ad390d702b16/torch/utils/data/graph_settings.py#L5 ?